packages/composites/augur/src/index.ts

```diff
@@ -1,7 +1,7 @@
 import { expose } from '@chainlink/ea-bootstrap'
 import { makeExecute } from './adapter'
-import { makeConfig } from './config'
+import { makeConfig, NAME } from './config'
 
-const NAME = 'AUGUR'
+const adapterContext = { name: NAME }
 
-export = { NAME, makeConfig, makeExecute, ...expose(NAME, makeExecute()) }
+export = { NAME, makeConfig, makeExecute, ...expose(adapterContext, makeExecute()) }
```

Comment:
Since we are changing this line, could we modify this to be standard with a standard export style?
```suggestion
const { server } = expose(adapterContext, makeExecute())
export { NAME, makeConfig, makeExecute, server }
```